TurfTests/TurfTests.swift

```diff
@@ -295,9 +295,10 @@ class TurfTests: XCTestCase {
            $0.map { CLLocationCoordinate2D(latitude: $0[1], longitude: $0[0]) }
         }
         let outerRing = Ring(coordinates: allRings.first!)
+        let innerRing = Ring(coordinates: allRings[1])
```

Comment:
`innerRings` can be expressed as `allRings.prefix(from: 1)` – handy for dealing with a two-hole polygon.

Reply:
It looks like `prefix(from:)` isn't available in Swift 4.

Reply:
Ooooo I think you meant `suffix(from: 1)` 😉

Reply:
Yeah that gets me every time. 😄